rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -645,6 +672,32 @@ def test_upgrade_v7_v8(globaldb: GlobalDBHandler):
             ('SPAM_ASSET_FALSE_POSITIVE', 'eip155:1/erc20:0xA0b73E1Ff0B80914AB6fe0444E65848C4C34450b'),  # noqa: E501
         ).fetchone()[0] == 1
 
+        # check that asset_collections have unique entries now with correctly mapped ids
+        assert table_exists(
+            cursor=cursor,
+            name='asset_collections',
+            schema=DB_CREATE_ASSET_COLLECTIONS,
+        ) is True
+
+        for collection_id, name, symbol in (
+            (260, 'Moon App', 'APP'),
+            (261, 'Starknet', 'STRK'),
+            (262, 'Wormhole Token', 'W'),
+            (263, 'TokenFi', 'TOKEN'),
+            (264, 'HTX', 'HTX'),
+            (300, 'Kyber Network Crystal v2', 'KNC'),
+            (301, 'PancakeSwap Token', 'CAKE'),
```

Comment:
is this the expected final state?

Reply:
yeah, they are the same values as in the update https://github.com/rotki/assets/blob/develop/updates/24/asset_collections_updates.sql
I didn't asserted all the values, only the starting and ending fews.